firefox-ios/Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -509,10 +509,8 @@ class TabManagerMiddleware {
         let windowManager: WindowManager = AppContainer.shared.resolve()
         guard uuid != .unavailable else {
             assertionFailure()
-            logger.log("Unexpected or unavailable UUID for TabManager. Returning active window tab manager by default.",
-                       level: .warning,
-                       category: .tabs)
-            return windowManager.tabManager(for: windowManager.activeWindow)
+            logger.log("Unexpected or unavailable window UUID for requested TabManager.", level: .fatal, category: .tabs)
```

Comment:
I'd like to 'upgrade' this log message in `TabManagerMiddleware` to a fatal. I don't currently know of any scenarios in which it can happen, but if it occurs in production I think we would want greater visibility. (cc @OrlaM @dataports)